app/client/cli/helpers/common.go

```diff
@@ -0,0 +1,14 @@
+package helpers
+
+import (
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/shared/modules"
+)
+
+var (
+	GenesisPath = runtime.GetEnv("GENESIS_PATH", "build/config/genesis.json")
```

Comment:
Add a TODO to:
1. Also accept reading this from datadir
2. Accept this is a flag